jetson/percep_obs_detect/include/find-clear-path.hpp

```diff
@@ -0,0 +1,48 @@
+#include "common.hpp"
+#include <float.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include "euclidean-cluster.hpp"
+//#include <cudamath>
+
+//TODO: where to put these constants in the final code
+
+static const float rovWidth = 1500; 
+
+static const int bearingNum = 1024; 
+
+static const float fov = 80; //Degree fov from straight ahead (0 degrees)
+
+class BearingLines{
+      public:
+        float heading; //Each bearingline defined by a heading
+        float2 n; //Normal vector to bearingline
+        float2 bLeft; //Left offset
+        float2 bRight; //Right offset
+
+        //Default Ctor
+        __device__ BearingLines();
+
+        //Ctor with heading in
+        __device__ BearingLines(float heading_in);
+};
+
+class FindClearPath {
+  public:
+
+    //Default Ctor
+    FindClearPath();
```

Comment:
add /brief statements for each function to keep consistent with the rest of the code